types/sinon-sandbox/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for sinon-sandbox 2.0.6
```

Comment:
```suggestion
// Type definitions for sinon-sandbox 2.0
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library